iphone/cli/commands/_build.js

```diff
@@ -3132,33 +3137,30 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 		},
 		legacySwift = version.lt(this.xcodeEnv.version, '8.0.0');
 
+	this.gccDefs = 'DEPLOYTYPE=' + this.deployType;
 	// set additional build settings
 	if (this.target === 'simulator' || this.target === 'macos') {
```

Comment:
Are these really build settings tied to the target or the deploy type? Seems like `DEBUG` should be true for `'development'` deploy type. The log id is used for writing our log file, so that seems tied to the target here. Not sure about `TI_VERSION`, but it looks to me like we set it explicitly in `writeXcodeConfigFiles()` anyways.

Reply:
These settings were already there. TI_VERSION is getting duplicate. So I removed it in my latest commit.